Middlewares/ProxyMiddleware.cs

```diff
@@ -0,0 +1,26 @@
+﻿public class ProxyMiddleware
+{
+    private readonly RequestDelegate _next;
+    private readonly HttpClient _httpClient;
+
+    public ProxyMiddleware(RequestDelegate next)
+    {
+        _next = next;
+        _httpClient = new HttpClient();
+    }
+
+    public async Task InvokeAsync(HttpContext context)
+    {
+        string remoteIpAddress = context.Connection.RemoteIpAddress?.ToString();
```

Comment:
What happen if RemoteIpAddress is null ?

Reply:
no exception will be thrown, unless you think it's important to handle it. Im open to any suggestion, but I forgot to mention this code is supposed to be temporary. This task is just the starting point for us to implement and perform new checks. 
Surely we will implement a better logging feature in the future this just shows how to access request information.